Code/GraphMol/Wrap/testThreads.py

```diff
@@ -34,12 +34,48 @@ def runner(func, args):
     
 
 nthreads = int(multiprocessing.cpu_count() * 100 / 4) # 100 threads per cpu
+threads = []
 for i in range(0, nthreads):
     for func in funcs:
         t = threading.Thread(target=runner, args=(func,core_mol))
         t.start()
+        threads.append(t)
     t = threading.Thread(target=runner, args=("ToBinary",None))        
     t.start()
+    threads.append(t)
+for t in threads:
+    t.join()
 
+# this spews a ton of logging info...
```

Comment:
This doesn't look like it's actually testing anything, just sending output to stdout. It may be interesting/useful for manual validation by the developer, but I don't think it should be part of the automated tests.